iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/TreeModelPlanner.java

```diff
@@ -46,12 +46,16 @@
 import org.apache.iotdb.rpc.RpcUtils;
 import org.apache.iotdb.rpc.TSStatusCode;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ScheduledExecutorService;
 
 public class TreeModelPlanner implements IPlanner {
+  private static final Logger LOGGER = LoggerFactory.getLogger(TreeModelPlanner.class);
```

Comment:
This can be removed

Reply:
Will be resolved in https://github.com/apache/iotdb/pull/12543/files